core/src/main/java/io/varhttp/CORSHandler.java

```diff
@@ -0,0 +1,70 @@
+package io.varhttp;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+
+public class CORSHandler {
+    private final CORSConfig config;
+
+    public CORSHandler(CORSConfig config) {
+        this.config = config;
```

Comment:
CORSHandlers should, like filters, be specific for a certain part of the configuration scope.

Reply:
this logic is no longer needed here. I will remove it.